### PR TITLE
Don't use shouty-caps in issue templates

### PR DESCRIPTION
**Pull request by johnkerl**
I'm having some buyer's remorse now that we're seeing these in issues -- I'd rather be looking at more lower-casing in the issues list.

Thoughts? :)

### CODE REVIEW

**Comment by johnkerl** [comment]
@maniarathi @atolopko-czi ping 🙏 -- accept, reject, request for revisions -- ?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[BUG]"
+title: "[bug]"
```
Comment (eddelbuettel) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: I still think 'Bug' is better in general English.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-title: "[FEATURE REQUEST]"
+title: "[feature request]"
```
Comment (eddelbuettel) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: I still think 'Feature Request' is better in general English.

**Review by eddelbuettel — APPROVED** [review]
:+1:

**Pull request merged**